<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QuatEditor</class>
 <widget class="QWidget" name="QuatEditor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>237</width>
    <height>38</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>W</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLineEdit" name="w_Edit"/>
   </item>
   <item row="0" column="2">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>X</string>
     </property>
    </widget>
   </item>
   <item row="0" column="3">
    <widget class="QLineEdit" name="x_Edit"/>
   </item>
   <item row="0" column="4">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Y</string>
     </property>
    </widget>
   </item>
   <item row="0" column="5">
    <widget class="QLineEdit" name="y_Edit"/>
   </item>
   <item row="0" column="6">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Z</string>
     </property>
    </widget>
   </item>
   <item row="0" column="7">
    <widget class="QLineEdit" name="z_Edit"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>w_Edit</sender>
   <signal>editingFinished()</signal>
   <receiver>QuatEditor</receiver>
   <slot>OnEditingFinished()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>73</x>
     <y>7</y>
    </hint>
    <hint type="destinationlabel">
     <x>71</x>
     <y>39</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>x_Edit</sender>
   <signal>editingFinished()</signal>
   <receiver>QuatEditor</receiver>
   <slot>OnEditingFinished()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>73</x>
     <y>27</y>
    </hint>
    <hint type="destinationlabel">
     <x>71</x>
     <y>39</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>y_Edit</sender>
   <signal>editingFinished()</signal>
   <receiver>QuatEditor</receiver>
   <slot>OnEditingFinished()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>73</x>
     <y>47</y>
    </hint>
    <hint type="destinationlabel">
     <x>71</x>
     <y>39</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>z_Edit</sender>
   <signal>editingFinished()</signal>
   <receiver>QuatEditor</receiver>
   <slot>OnEditingFinished()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>73</x>
     <y>67</y>
    </hint>
    <hint type="destinationlabel">
     <x>71</x>
     <y>39</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>OnEditingFinished()</slot>
 </slots>
</ui>
